system/keymaps/touchscreen.xml

```diff
@@ -84,8 +84,8 @@
     <touch>
       <zoom>ZoomGesture</zoom>
       <rotate>RotateGesture</rotate>
-      <swipe direction="right" pointers="2">PreviousPicture</swipe> <!-- right/left are flipped here to make picture flipping feel more natural on touch screens. Two pointers are used in order to avoid conflicting with the panning gesture on zoomed-in images. -->
-      <swipe direction="left" pointers="2">NextPicture</swipe>
+      <swipe direction="right">PreviousPicture</swipe> <!-- right/left are flipped here to make picture flipping feel more natural on touch screens. Two pointers are used in order to avoid conflicting with the panning gesture on zoomed-in images. -->
```

Comment:
Thx for the hint - there is indeed a conflict ... will think About something else ...